management.md

```diff
@@ -7,6 +7,34 @@
 
 - How will you divide the entire team into sub-teams assigned to complete
   specific tasks?
+  - Specify tasks that need to be completed
+
+    When dividing the entire team into sub-teams, the first step is realizing all
+    the tasks that need to be accomplished. This includes making an expansive list
+    of tasks and taking care not to overlook details even if they seem unimportant.
+
+  - Divide tasks into categories of labor
+
+    Once a comprehensive list of tasks exists, the next step is to divide these
+    tasks according to category of labor. The distribution of tasks should be as
+    even as possible while remaining accurate.
+
+  - Divide team into categories of labor
+    Once tasks have been divided accordingly, it will be possible to divide the
+    team into sub-teams to complete tasks. Each sub-team should contain members
+    with different skills so that groups can divide their assigned labor accordingly.
+    For example, each group should have a `Project Manager` that is able to oversee
+    group operations and communicate with other group managers. This will allow
+    for lateral communication and provide each group with the power of the entire
+    team through the responsibility of each person to communicate outside the group
+    according to their skillset.
```

Comment:
This part is not clear. I'm not sure if I understand "provide each group with the power of the entire team". Is everyone in the team responsible for communicating with other groups or just Project Managers?